proto/utils/v1beta1/custom_options.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package utils.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/utils";
+
+import "google/protobuf/descriptor.proto";
+import "permission/exported/v1beta1/types.proto";
+
+extend google.protobuf.MessageOptions {
+  permission.exported.v1beta1.Role permission_role =
```

Comment:
Interesting. Are you going to add the permission to each message proto in a different PR?

Reply:
In this PR, I just wanted to show you how the test looks when it fails

Reply:
Yeah, just realized, thanks